src/Sylius/Behat/Context/Transform/CurrencyContext.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Currency\Converter\CurrencyNameConverterInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+final class CurrencyContext implements Context
+{
+    /**
+     * @var CurrencyNameConverterInterface
+     */
+    private $converter;
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $repository;
```

Comment:
We should use full names as we do everywhere else:
* `converter` -> `currencyNameConverter`
* `repository` -> `currencyRepository`